app/src/modules/settings/routes/data-model/field-detail/shared/extension-options.vue

```diff
@@ -70,7 +70,10 @@ export default defineComponent({
 
 		const { collection, field } = storeToRefs(fieldDetailStore);
 
-		const extensionInfo = useExtension(props.type, props.extension);
+		const extensionInfo = useExtension(
+			props.type,
```

Comment:
Won't this have the same issue if the type of extension changes?

Reply:
According to current implementation, `extension-options` component receives static string from parent. Definitely in future, we can update `type` to reactive.